src/qt/bitcoingui.cpp

```diff
@@ -104,7 +108,7 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *platformStyle, const NetworkStyle *n
 {
     GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
 
-    QString windowTitle = tr("Bitcoin Core") + " - ";
+    QString windowTitle = tr(PACKAGE_NAME) + " - ";
```

Comment:
I believe this has been addressed [here](https://github.com/bitcoin/bitcoin/pull/7192/files#diff-7dbd14e796e6175513dfdc05c68a5b39R73).